sdk/cmake/msvc.cmake

```diff
@@ -448,37 +432,7 @@ function(allow_warnings __module)
 endfunction()
 
 macro(add_asm_files _target)
-    if(MSVC_IDE)
```

Comment:
Is the Visual Studio solutions generator now fixed in CMake, so that it allows VS solutions to correctly compile ASM files as well?

Reply:
Apparently not. I reverted this, will look into it later.